src/github_changelog/github.clj

```diff
@@ -15,13 +15,16 @@
                   :or   {github-api (:github-api defaults/config)}}]
   (format "%s/repos/%s/%s/pulls" (util/strip-trailing github-api) user repo))
 
+(defn- auth-headers [token]
+  {"User-Agent"    "GitHub-Changelog"
```

Comment:
Please address @mszabo's suggestion by always sending out our User-Agent so that tooling can block us if necessary.